src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMBuilderTest.java

```diff
@@ -2600,8 +2600,8 @@ public void given__server_pullMerge_norev_userkey__when__build__then__scmBuilt()
                 new BranchSCMHead("test-branch"),
                 new SCMHeadOrigin.Fork("qa/qa-repo"),
                 ChangeRequestCheckoutStrategy.MERGE);
-        source.setCredentialsId("user-key");
         GitHubSCMBuilder instance = new GitHubSCMBuilder(source, head, null);
+        instance.withCredentials("user-key", GitHubSCMBuilder.SSH);
```

Comment:
Nit: prefer non deprecated API usage
```suggestion
        GitHubSCMBuilder instance = new GitHubSCMBuilder(source, head, null)
                .withCredentials("user-key")
                .withResolver(GitHubSCMBuilder.SSH);
```